test/support/rekor_helper.rb

```diff
@@ -1,7 +1,12 @@
+require 'rubygems/sigstore/gemfile'
+require 'rubygems/sigstore/crypto'
+
 module RekorHelper
   include UrlHelper
+  include FulcioHelper
 
   REKOR_BASE_URL = 'https://rekor.sigstore.dev/'
+  FAKE_CA_BASE_URL = 'http://some-ca-authority.org/'
```

Comment:
\nit I would maybe prefix `FAKE_CA_BASE_URL` with `REKOR` to keep the namespace a bit tidier where the module gets included. What do you think?